Automattic/jetpack
# Add Newsletter Post Publish Panel

Fixes https://github.com/Automattic/gold/issues/34

## Proposed changes:

Adds the Post Publish Panel for newsletters based on the designs

| Design  | Implementation |
| ------------- | ------------- |
| <img width="281" alt="Screenshot 2023-05-11 at 6 08 29 PM" src="https://github.com/Automattic/jetpack/assets/7076981/6d329fbc-f604-41a7-a941-a15c682df3c2"> | <img width="273" alt="Screenshot 2023-05-11 at 6 15 35 PM" src="https://github.com/Automattic/jetpack/assets/7076981/1152ed68-bea7-41cc-be26-cff2aeddcd72">  |



### Other information:





## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Sync with WPCOM
* Ensure `.config/jetpack-loader.php` is on `production`
* Create a new post and verify the Newsletter post publish panel matches the designs
